packages/@sanity/cli/src/types.ts

```diff
@@ -310,6 +310,8 @@ export interface CliConfig {
   graphql?: GraphQLAPIConfig[]
 
   vite?: UserViteConfig
+
+  enableAutoUpdates?: boolean
```

Comment:
Should this be `unstable_enableAutoUpdates` ? We don't want this being used at the moment